src/Surfnet/StepupRa/RaBundle/Resources/public/js/stepup-ra.js

```diff
@@ -188,5 +188,11 @@
             $('button#ra_search_ra_second_factors_export').hide();
         }
 
+        // Post sub institution switcher on change
+        $(document).on('change', '#select_institution_institution', function() {
```

Comment:
Got to love the jQuery! :heart_eyes: Haven't seen that in a while..

On a more serious note, is listening for 'change' the expected behavior, or is 'blur' more user friendly? I'm no UX expert by any means.. Just thinking out loud here. I personally prefer 'on change'.

Reply:
The `change` event is the one you'd like else if you get focus and you lose it your page is getting refreshed even if the value isn't changed at all.